pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -215,6 +215,16 @@ func (m *kubeGenericRuntimeManager) calculateLinuxResources(cpuRequest, cpuLimit
 		resources.CpuPeriod = cpuPeriod
 	}
 
+	// the "memory.oom.group" flag appeared in kernel 4.19 and the docs indicate that 5.8 is the minimum kernel version
+	// if using cgroups v2, so we just need to check for that
+	if libcontainercgroups.IsCgroup2UnifiedMode() {
+		// Turn on memory.oom.group which tells the kernel to treat the entire container cgroup as
+		// a single unit for OOM kill purposes.  This will cause all processes in the container to be
+		// OOM killed together if any are OOM killed.
```

Comment:
nit: this comment belongs to `"memory.oom.group": "1"` not to the `resources.Unified =` line.

You can also make the comment more concise; something like

```
// Ask the kernel to kill all processes in the container cgroup in case of OOM.
```